test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/V3DetailControlModelTests.cs

```diff
@@ -131,6 +131,20 @@ public void VulnerabilityCountWhenMetadataHasVulnerability_Calculated()
             Assert.Equal(_testInstance.PackageVulnerabilityCount, _testData.TestData.Vulnerabilities.Count());
         }
 
+        [Fact]
+        public void PackageVulnerabilitiesWhenMetadataHasVulnerability_OrderedBySeverityDescending()
+        {
+            Assert.Collection(_testInstance.PackageVulnerabilities,
```

Comment:
This test takes a dependency on `V3PackageSearchMetadataFixture`'s ctor, so changes there may break this test.
Our general preference for testing is to include a minimal setup in the Arrange of each test that makes it clear what the input data is, and the expected output. Another benefit of that is you can easily test 2-3 combinations instead of just 1. 
Would it be difficult to continue this pattern for this test?

Reply:
Thanks for pointing this out. I agree with you that hard-coding the expected values wasn't great since it could break if the TestData were changed. Like the other tests in this test class, I kept the dependency on the fixture but I changed it now to set up the TestData as it's expected to be and compare the actual results to it. This would allow the same test to work against different test data in the future if we parameterize the testdata and the test instance.